### PR TITLE
bump qt minimum version to 5.9

### LINKED ISSUE #5331: Qt 5.6 LTS ended in march 2019
https://www.qt.io/blog/2018/11/29/support-qt-5-6-lts-ends-march-2019

| Qt version | LTS till |
| :------ | :------- |
| 5.9 | 31.05.2020 |
| 5.12 | ??.??.2021 |

I'll suggest to update `CMakeLists.txt` to set the recommended version to atleast version `5.9`

### PR DESCRIPTION
**Pull request by Rossmaxx**
fixes #5331.

### CODE REVIEW

**Review by messmerd — APPROVED** [review]

**Comment by DomClark** [comment]
Now the minimum Qt version has increased, you could additionally clean up some obsolete compatibility code for older Qt versions (search for `QT_VERSION`). This PR is also fine as it is, though.

**Comment by Rossmaxx** [comment]
@DomClark done.

File: src/gui/MainWindow.cpp [hunk 1]
```diff
@@ -325,7 +325,7 @@ void MainWindow::finalize()
 					Qt::CTRL + Qt::Key_M );
 
 // Prevent dangling separator at end of menu per https://bugreports.qt.io/browse/QTBUG-40071
-#if !(defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600))
+#ifndef LMMS_BUILD_APPLE
```
Comment (Rossmaxx) [inline src/gui/MainWindow.cpp:328]: I have small doubt about this one line tho.

Comment (DomClark) [reply src/gui/MainWindow.cpp:328]: This check can be removed entirely. `QT_VERSION < 0x050600` is always false now, so `defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600)` is always false too, and `!(defined(LMMS_BUILD_APPLE) && (QT_VERSION < 0x050600))` is always true.

Comment (Rossmaxx) [reply src/gui/MainWindow.cpp:328]: Ok

Comment (Rossmaxx) [reply src/gui/MainWindow.cpp]: Removed the check.

**Review by DomClark — APPROVED** [review]

**Pull request merged**